jmeter/plugins/jmeter-backendlistener-maestro/src/test/java/com/farfetch/jmeter/backendlistener/maestro/TestMaestroBackendClient.java

```diff
@@ -0,0 +1,6 @@
+package com.farfetch.jmeter.backendlistener.maestro;
+
+// WIP
+public class TestMaestroBackendClient {
```

Comment:
not sure if you have any plans to add tests here, we can remove it if not. 

I think, would be good to cover up only some core logic like mapping metrics to the proper format.

We could do this as the last step once have everything else done and configured.

Reply:
Addressed on PR #204